egs/wenetspeech/ASR/pruned_transducer_stateless2/train.py

```diff
@@ -957,6 +961,34 @@ def remove_short_and_long_utt(c: Cut):
         torch.distributed.barrier()
         cleanup_dist()
 
+def display_and_save_batch(
+    batch: dict,
+    params: AttributeDict,
+) -> None:
+    """Display the batch statistics and save the batch into disk.
+
+    Args:
+      batch:
+        A batch of data. See `lhotse.dataset.K2SpeechRecognitionDataset()`
+        for the content in it.
+      params:
+        Parameters for training. See :func:`get_params`.
+      sp:
+        The BPE model.
```

Comment:
```suggestion
```

Reply:
I see, I have modified num_tokens in display_and_save_batch